animations/index.md

```diff
@@ -1,159 +1,161 @@
+     <td style="width:5%">
+     <a href="" onMouseOver="document.MyImage1.src='/animations/images/animated_opacity.gif';" onMouseOut="document.MyImage1.src='/animations/images/animated_opacity.png';">
+     <img src="/animations/images/animated_opacity.png" name="MyImage1" height="180" width="160">
+     </a></td>
+   </tr>
+ </table>
+
+
+
+ <br>
+ For more information including a list of implicit widgets, see [Flutter Implicit Animations](/animations/implicit_widgets).
+
+
+### Transition animation widgets
+
+*Transition animations* are easy to use like implicit animations because transition animation widgets are pre-defined for you, but they also allow you to customize some elements that are included in the explicit animations. With transition animation widgets, you can specify a visual effect that determines how the animation target object is animated. You include the `AnimationController` to control the animation. For example, you can control how the object is made visible or hidden or how to get from the start value to the end value.  
+
+ <table cellpadding="5" style="width:650px">
+  <tr>
+   <td style="width:5%">
+  <a href="" onMouseOver="document.MyImage3.src='/animations/images/rotation_transition.gif';" onMouseOut="document.MyImage3.src='/animations/images/rotation_transition.png';">
+  <img src="/animations/images/rotation_transition.png" name="MyImage3" height="180" width="160" name="MyImage3">
+  </a></td>
+    <td style="width:15%">
+   The <code>RotationTransition</code> and <code>SlideTransition</code> widgets are two animation examples that use transition animation widgets.
+    </td>
+    <td style="width:5%">
+    <a href="" onMouseOver="document.MyImage4.src='/animations/images/slide_transition.gif';" onMouseOut="document.MyImage4.src='/animations/images/slide_transition.png';">
+    <img src="/animations/images/slide_transition.png" name="MyImage4" height="180" width="160">
+    </a></td>
+  </tr>
+</table>
+
+
+
+<br>
+For more information including a list of transition widgets, [Flutter Transition Animations](/animations/transition_widgets).  
+
+
+### Explicit animation widgets
+
+ *Explicit animations* involve building customized animations using the `AnimationBuilder` widget that allows you to customize many of the animation elements that are pre-defined in the implicit and transition animation widgets. When you build explicit animations, you manually add listeners, tickers, and other elements for customization.  
+ <br>
+ For more information, see [Flutter Explicit Animations](/animations/explicit_widgets).  
+
+## Choosing an animation widget
+Choosing an animation widget depends on the customization that you need for your animation and whether there is an existing widget available for reuse.
+
+* Choose an implicit animation when you want to reuse an existing implicit animation widget and change the animation object. Using an implicit animation widget allows you to set the duration, and start and end points.
```

Comment:
good additions - I'm updating the brief description here/on the intro page, and then adding more detail to the implicit animations page so that we don't get to detailed on this intro page

one thing we want to be careful of is providing too much detail on the intro page which should function as a lead-in to the other specific pages describing the animation widgets in much more detail